tests/pw/package.json

```diff
@@ -68,6 +69,7 @@
         "dotenv": "^16.4.5",
         "mysql2": "^3.11.0",
         "php-serialize": "^5.0.1",
+        "randomstring": "^1.3.0",
```

Comment:
Instead of using radomstring package. use Use faker.string.alpha() from faker-js package which is already available